integration/app/app.component.ts

```diff
@@ -39,10 +38,10 @@ export class AppComponent implements OnInit {
   });
 
   greeting: string;
-  todos$: Observable<Todo[]> = this._store.select(TodoState);
-  pandas$: Observable<Todo[]> = this._store.select(TodoState.getPandas);
-  pizza$: Observable<Pizza> = this._store.select(TodosState.getPizza);
-  injected$: Observable<string> = this._store.select(TodosState.getInjected);
+  todos: Signal<Todo[]> = this._store.selectSignal(TodoState);
+  pandas: Signal<Todo[]> = this._store.selectSignal(TodoState.getPandas);
+  pizza: Signal<Pizza> = this._store.selectSignal(TodosState.getPizza);
+  injected: Signal<string> = this._store.selectSignal(TodosState.getInjected);
```

Comment:
This comment refers to all the changes in the integration app...
We should have a signal and an observable variant in the implementations because both usages are still valid scenarios.